test/src/index-test.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import path from 'path';
+import stardust from 'index';
+
+let componentPaths = require.context('src/components/', true, /[^Style\.js$]\.js$/i);
+let componentNames = _.map(componentPaths.keys(), key => {
+  return path.basename(key).replace('.js', '');
+});
+
+describe('index.js', () => {
+  it('has a property for every component', () => {
+    _.each(componentNames, component => {
+      expect(component in stardust);
+      expect(stardust[component]).to.be.a('function');
+    });
+  });
```

Comment:
Here we test that all components in the components dir are keys on `index.js` and that their values are functions.